cmd/slsa-github-generator/provenance.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"encoding/json"
+	"errors"
+
+	"github.com/spf13/cobra"
+
+	"github.com/slsa-framework/slsa-github-generator/github"
+	"github.com/slsa-framework/slsa-github-generator/slsa"
+)
+
+// provenanceCmd returns the 'provenance' command.
+func provenanceCmd() *cobra.Command {
```

Comment:
It's was a command I thought might be useful to generate provenance without going though the full signing process for those who want to test or play around with the project.

I can remove it if it's truly not necessary but I was inclined to add it as a separate command since it didn't really fit well as an option on the attest command.

Reply:
Yeah, it is maybe confusing. I think we can just remove this and give some extra instructions for debugging. 

It's a bit difficult to set up and run because you need a github context and OIDC provider so it's only really practical to run if running in a github action, but you have to kind of fake the github context for this command anyway so it's probably not really worth it.

I'll track that in #6

Reply:
Removed.